portal/src/main/webapp/js/src/oncoprint/studyview.js

```diff
@@ -5,10 +5,94 @@ $(document).ready(function() {
 		interpolate: /\{\{(.+?)\}\}/g
 	};
 	$('#oncoprint_controls').html(_.template($('#main-controls-template').html())());
+        
+        var clinicalElementsArray = [];
+        var GenePanelData = [];
+        var genePanelClinicals = new ClinicalColl();
+        clinicalElementsArray.push(
+        genePanelClinicals.fetch({
+        type: "POST",
+        data: {
+           cancer_study_id: cancer_study_id_selected,
+           attribute_id: "GENE_PANEL",
+           case_list: cases
+        },
+        success: function(response) {
+               GenePanelData = response.toJSON();
+           }
+        }));
+        
+        //fetch genepanel
+        var genepanelFiles = ["api/genepanel/IMPACT341","api/genepanel/IMPACT410"];
+        var genePanel = new Array();
+        var gainGenepanel = function(filename,datafetchArray)
```

Comment:
yes. You are right,  you can change to the right word

On Thu, Sep 17, 2015 at 1:24 PM, adamabeshouse <notifications@github.com>
wrote:

> In portal/src/main/webapp/js/src/oncoprint/studyview.js
> <https://github.com/cBioPortal/cbioportal/pull/254#discussion_r39773192>:
>
> > +        genePanelClinicals.fetch({
> > +        type: "POST",
> > +        data: {
> > +           cancer_study_id: cancer_study_id_selected,
> > +           attribute_id: "GENE_PANEL",
> > +           case_list: cases
> > +        },
> > +        success: function(response) {
> > +               GenePanelData = response.toJSON();
> > +           }
> > +        }));
> > +
> > +        //fetch genepanel
> > +        var genepanelFiles = ["api/genepanel/IMPACT341","api/genepanel/IMPACT410"];
> > +        var genePanel = new Array();
> > +        var gainGenepanel = function(filename,datafetchArray)
>
> gain isn't the right word. maybe fetch?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cBioPortal/cbioportal/pull/254/files#r39773192>.
>